zorg/buildbot/commands/LitTestCommand.py

```diff
@@ -16,6 +16,8 @@ class LitLogObserver(LogLineObserver):
   # output always immediately follow a test.
   kTestVerboseLogStartRE = re.compile(r"""\*{4,80} TEST '(.*)' .*""")
   kTestVerboseLogStopRE = re.compile(r"""\*{10,80}""")
+  kTestVerboseGTestUnitRE = re.compile(r'^(.*)\/(\d*)\/(\d*)$')
+  kTestVerboseGTestPathRE = re.compile(r'^(.+) --gtest_filter=(\w+).(\w+)$')
```

Comment:
`^(.+)\s+--gtest_filter=(\w+).(\w+)[\s$]` will not work because `[\s$]` means a space or $ sign, not the EOL.

I will update the regex with the second suggestion.

Reply:
`\s$` - space or EOL
`\s\$` - space or `$` sign
but anyway, I'm ok with any of those regexes.

it should work. I have tested it here https://regex101.com/

Reply:
https://regex101.com/r/m187iA/1

Reply:
You didn't add EOL to the test string there; otherwise it gets matched.

Reply:
Note `^(.+)\s+--gtest_filter=(\w+).(\w+)(\s|$)` or even `^(.+)\s+--gtest_filter=(\w+).(\w+)$` work without `\n` at the end.

Anyway outLineReceived(self, line) receives the line without `\n` at the end. Look at _lineReceived() in buildbot/master/buildbot/process/logobserver.py